tests/utils/datavolume.go

```diff
@@ -36,6 +36,8 @@ const (
 	InvalidQcowImagesURL = "http://cdi-file-host.%s/invalid_qcow_images/"
 	// TarArchiveURL provides a test url for a tar achive file
 	TarArchiveURL = "http://cdi-file-host.%s/archive.tar"
+	// CirrosExternalURL provides the standard cirros image download url external from our test env
+	CirrosExternalURL = "http://download.cirros-cloud.net/0.4.0/cirros-0.4.0-x86_64-disk.img"
```

Comment:
I don't think I want to depend on an external source. from CDIs perspective there is no difference in going to the internet and going to an http server hosted in the cluster (besides maybe which network to go on). We already have the cirros image in the repo tests/images/cirros.qcow2 we just need to copy it into the http container in the right directory. /tmp/shared/images

Reply:
No problem, I can change that if you think it's a safe assumption.